.github/CODEOWNERS

```diff
@@ -0,0 +1,81 @@
+/src/adapter                        @MaterializeInc/surfaces
+/src/alloc                          @benesch
+/src/audit-log                      @MaterializeInc/surfaces
+/src/avro                           @umanwizard
+/src/avro-derive                    @umanwizard
+/src/build-id                       @umanwizard
+/src/build-info                     @benesch
+/src/ccsr                           @benesch
+/src/cloud-resources                @MaterializeInc/cloud
+/src/clusterd                       @MaterializeInc/compute @MaterializeInc/storage
+/src/compute                        @MaterializeInc/compute
+/src/compute-client                 @MaterializeInc/compute
+/src/environmentd                   @MaterializeInc/surfaces
+/src/expr                           @MaterializeInc/compute
+/src/expr-test-util                 @MaterializeInc/compute
+/src/frontegg-auth                  @MaterializeInc/surfaces
+/src/http-util                      @MaterializeInc/surfaces
+/src/interchange                    @MaterializeInc/storage
+/src/kafka-util                     @MaterializeInc/storage
+/src/kinesis-util                   @MaterializeInc/storage
+/src/lowertest                      @MaterializeInc/compute
+/src/lowertest-derive               @MaterializeInc/compute
+/src/metabase                       @benesch
+/src/mz                             @MaterializeInc/surfaces
+/src/npm                            @benesch
+/src/orchestrator                   @benesch
+/src/orchestrator-kubernetes        @benesch
+/src/orchestrator-process           @benesch
+/src/orchestrator-tracing           @benesch
+/src/ore                            @benesch
+/src/persist                        @MaterializeInc/persist
+/src/persist-client                 @MaterializeInc/persist
+/src/persist-types                  @MaterializeInc/persist
+/src/pgcopy                         @MaterializeInc/surfaces
+/src/pgrepr                         @MaterializeInc/surfaces
+/src/pgtest                         @MaterializeInc/surfaces
+/src/pgwire                         @MaterializeInc/surfaces
+/src/pid-file                       @benesch
+/src/postgres-util                  @MaterializeInc/surfaces
+/src/prof                           @umanwizard
+/src/proto                          @aalexandrov
+/src/repr                           @MaterializeInc/storage @MaterializeInc/compute
+/src/repr/src/row                   @MaterializeInc/persist
+/src/repr-test-util                 @MaterializeInc/compute
+/src/s3-datagen                     @MaterializeInc/storage
+/src/secrets                        @MaterializeInc/cloud
+/src/segment                        @benesch
+/src/service                        @MaterializeInc/storage @MaterializeInc/compute
+/src/sql                            @MaterializeInc/surfaces
+/src/sql-parser                     @MaterializeInc/surfaces
+/src/sqllogictest                   @MaterializeInc/surfaces
```

Comment:
Should QA joint-own all our testing infra, including co-owning `/src/sqllogictest` and `/src/pgtest` with surfaces?

Reply:
cc @philip-stoev